ground/src/main/java/com/google/android/ground/model/mutation/SubmissionMutation.kt

```diff
@@ -56,14 +56,6 @@ data class SubmissionMutation(
     var responseDeltas: ImmutableList<ResponseDelta> = ImmutableList.of()
       @JvmSynthetic set
 
-    fun setJob(job: Job): Builder = apply { this.job = job }
-
-    fun setSubmissionId(id: String): Builder = apply { this.submissionId = id }
-
-    fun setResponseDeltas(deltas: ImmutableList<ResponseDelta>): Builder = apply {
-      this.responseDeltas = deltas
-    }
-
     override fun build(): SubmissionMutation =
```

Comment:
Can we remove the `Builder` class entirely at this point?

Reply:
There is one usage left which is in Java. So, we need to convert it to Kotlin and then perform the same steps to get rid of Builder